pkg/services/auth/auth_token.go

```diff
@@ -404,6 +410,49 @@ func (s *UserAuthTokenService) rotatedAfterParam() int64 {
 	return getTime().Add(-s.Cfg.LoginMaxInactiveLifetime).Unix()
 }
 
+const (
+	concurrentSessionLimit    = 3
+	concurrentSessionLimitKey = "concurrentSessionLimit"
+)
+
+func (s *UserAuthTokenService) isConcurrentSessionLimitEnabled() bool {
+	return s.Cfg.FeatureToggles[concurrentSessionLimitKey]
+}
+
+func (s *UserAuthTokenService) revokeOldUserTokens(ctx context.Context, userId int64) error {
+	if !s.isConcurrentSessionLimitEnabled() {
```

Comment:
Isn't better to check this in `CreateToken()` function? In that case it looks more straightforward and clear I think.